apps/aechannel/src/aesc_fsm.erl

```diff
@@ -2793,12 +2790,12 @@ send_recoverable_error_msg(?WDRAW_ERR, Sn, Msg) ->
     lager:debug("send withdraw_error: ~p", [Msg]),
     aesc_session_noise:wdraw_error(Sn,Msg);
 send_recoverable_error_msg(?SHUTDOWN_ERR, Sn, Msg) ->
-    case aesc_offchain_update:get_vsn() of
-        V when V > 1 ->
+    case was_fork_activated(?LIMA_PROTOCOL_VSN) of
```

Comment:
Taking advantage of this being Lima already. We could also remove the other head as well

Reply:
Actually since we dropped the support for older FSMs and don't expect syncing with them, this whole case is not necessary